src/docfx/Errors.cs

```diff
@@ -373,16 +373,16 @@ public static Error ViolateSchema(SourceInfo source, string message)
         /// <summary>
         /// The input value type does not match expected value type.
         /// </summary>
-        /// Behavior: ✔️ Message: ❌
+        /// Behavior: ✔️ Message: ✔️
         public static Error UnexpectedType(SourceInfo source, object expectedType, object actualType)
-            => new Error(ErrorLevel.Warning, "unexpected-type", $"Expect type '{expectedType}' but got '{actualType}'", source);
+            => new Error(ErrorLevel.Warning, "unexpected-type", $"Expected type '{expectedType}' but got '{actualType}'", source);
```

Comment:
The by default ErrorLevel for UnexpectedType in docfx is warning? Shall we override it using ErrorLevel in schema?

Reply:
good idea